NOTE: Patch slightly modified from original; unmodified comments from 
original patch copied below.
=============================================================================
http://bugs.openembedded.net/show_bug.cgi?id=2617
- Fixes default ALSA state for SL-Cxx00.
  * Treble level reflects kernel fix alsa-wm8750-treble.patch (>2.6.24).
  * Increased Mic Levels and Boost.
  * Enabled Capture Switch.
  * Improved Headphones / Speaker volume balance.
  * 3D Mode changed to "Playback" (still Off by default).
  * Disabled all bypasses.
  * All Mono signals decreased to 0 (not connected).
  * Line Mux changed to differential, keeping PGA unused.
  * Using Digital mono for recording (jack supports ony mono).
  * Enabled Automatic Level Control, optimized for voice recording.
  * Enabled Zero Cross volume changes.
- Do not alter incorrect mixer levels by zaurus-mixer-callback.

--- zaurusd/config/alsa/cxx00-default.state	2009-04-25 22:57:30.000000000 +0100
+++ zaurusd/config/alsa/cxx00-default.state	2009-05-14 10:41:29.000000000 +0100
@@ -6,8 +6,8 @@
 		comment.range '0 - 63'
 		iface MIXER
 		name 'Capture Volume'
-		value.0 23
-		value.1 23
+		value.0 48
+		value.1 48
 	}
 	control.2 {
 		comment.access 'read write'
@@ -15,8 +15,8 @@
 		comment.count 2
 		iface MIXER
 		name 'Capture ZC Switch'
-		value.0 false
-		value.1 false
+		value.0 true
+		value.1 true
 	}
 	control.3 {
 		comment.access 'read write'
@@ -24,8 +24,8 @@
 		comment.count 2
 		iface MIXER
 		name 'Capture Switch'
-		value.0 false
-		value.1 false
+		value.0 true
+		value.1 true
 	}
 	control.4 {
 		comment.access 'read write'
@@ -33,8 +33,8 @@
 		comment.count 2
 		iface MIXER
 		name 'Headphone Playback ZC Switch'
-		value.0 false
-		value.1 false
+		value.0 true
+		value.1 true
 	}
 	control.5 {
 		comment.access 'read write'
@@ -42,8 +42,8 @@
 		comment.count 2
 		iface MIXER
 		name 'Speaker Playback ZC Switch'
-		value.0 false
-		value.1 false
+		value.0 true
+		value.1 true
 	}
 	control.6 {
 		comment.access 'read write'
@@ -92,8 +92,8 @@
 		comment.range '0 - 255'
 		iface MIXER
 		name 'PCM Volume'
-		value.0 255
-		value.1 255
+		value.0 220
+		value.1 220
 	}
 	control.11 {
 		comment.access 'read write'
@@ -131,7 +131,7 @@
 		comment.range '0 - 15'
 		iface MIXER
 		name 'Treble Volume'
-		value 15
+		value 0
 	}
 	control.15 {
 		comment.access 'read write'
@@ -188,7 +188,7 @@
 		comment.item.1 Playback
 		iface MIXER
 		name '3D Mode'
-		value Capture
+		value Playback
 	}
 	control.21 {
 		comment.access 'read write'
@@ -226,7 +226,7 @@
 		comment.count 1
 		iface MIXER
 		name 'ALC Capture ZC Switch'
-		value false
+		value true
 	}
 	control.25 {
 		comment.access 'read write'
@@ -235,7 +235,7 @@
 		comment.range '0 - 15'
 		iface MIXER
 		name 'ALC Capture Hold Time'
-		value 0
+		value 9
 	}
 	control.26 {
 		comment.access 'read write'
@@ -244,7 +244,7 @@
 		comment.range '0 - 15'
 		iface MIXER
 		name 'ALC Capture Decay Time'
-		value 3
+		value 9
 	}
 	control.27 {
 		comment.access 'read write'
@@ -253,7 +253,7 @@
 		comment.range '0 - 15'
 		iface MIXER
 		name 'ALC Capture Attack Time'
-		value 2
+		value 3
 	}
 	control.28 {
 		comment.access 'read write'
@@ -306,7 +306,7 @@
 		comment.count 1
 		iface MIXER
 		name 'ZC Timeout Switch'
-		value false
+		value true
 	}
 	control.34 {
 		comment.access 'read write'
@@ -331,8 +331,8 @@
 		comment.range '0 - 3'
 		iface MIXER
 		name 'Mic Boost'
-		value.0 2
-		value.1 2
+		value.0 3
+		value.1 3
 	}
 	control.37 {
 		comment.access 'read write'
@@ -341,8 +341,8 @@
 		comment.range '0 - 7'
 		iface MIXER
 		name 'Bypass Left Playback Volume'
-		value.0 2
-		value.1 2
+		value.0 0
+		value.1 0
 	}
 	control.38 {
 		comment.access 'read write'
@@ -361,8 +361,8 @@
 		comment.range '0 - 7'
 		iface MIXER
 		name 'Bypass Mono Playback Volume'
-		value.0 2
-		value.1 2
+		value.0 0
+		value.1 0
 	}
 	control.40 {
 		comment.access 'read write'
@@ -379,8 +379,8 @@
 		comment.range '0 - 127'
 		iface MIXER
 		name 'Headphone Playback Volume'
-		value.0 70
-		value.1 70
+		value.0 105
+		value.1 105
 	}
 	control.42 {
 		comment.access 'read write'
@@ -389,8 +389,8 @@
 		comment.range '0 - 127'
 		iface MIXER
 		name 'Speaker Playback Volume'
-		value.0 70
-		value.1 70
+		value.0 127
+		value.1 127
 	}
 	control.43 {
 		comment.access 'read write'
@@ -399,7 +399,7 @@
 		comment.range '0 - 127'
 		iface MIXER
 		name 'Mono Playback Volume'
-		value 121
+		value 0
 	}
 	control.44 {
 		comment.access 'read write'
@@ -411,7 +411,7 @@
 		comment.item.3 'Digital Mono'
 		iface MIXER
 		name 'Right ADC Mux'
-		value 'Mono (Left)'
+		value 'Digital Mono'
 	}
 	control.45 {
 		comment.access 'read write'
@@ -423,7 +423,7 @@
 		comment.item.3 'Digital Mono'
 		iface MIXER
 		name 'Left ADC Mux'
-		value 'Mono (Left)'
+		value 'Digital Mono'
 	}
 	control.46 {
 		comment.access 'read write'
@@ -471,7 +471,7 @@
 		comment.item.4 Differential
 		iface MIXER
 		name 'Left Line Mux'
-		value PGA
+		value Differential
 	}
 	control.50 {
 		comment.access 'read write'
